lib/src/lifecycle_module.dart

```diff
@@ -487,7 +487,7 @@ abstract class LifecycleModule extends SimpleModule with Disposable {
   /// Attempting to load a child module after a module has been unloaded will
   /// throw a [StateError].
   @protected
-  Future<Null> loadChildModule(LifecycleModule childModule) {
+  Future<Null> loadChildModule(LifecycleModule? childModule) {
     if (isOrWillBeDisposed) {
       return _buildDisposedOrDisposingResponse(methodName: 'loadChildModule');
     }
```

Comment:
If you update line 495 to also return if the childModule is null, then you can remove the following `!` on it.
```
    if (childModule == null || _childModules.contains(childModule)) {
      return Future.value(null);
    }
```